src/Frontend/Assets.php

```diff
@@ -173,6 +186,22 @@ protected function makeLessCompiler(string $filename): LessCompiler
             $compiler->setFileSourceOverrides($this->fileSourceOverrides);
         }
 
+        /** @var SettingsRepositoryInterface */
+        $settings = resolve(SettingsRepositoryInterface::class);
+
+        $coreFunctions = [
+            'is-extension-enabled' => function (Less_Tree_Quoted $extensionId) use ($settings) {
+                $raw = $settings->get('extensions_enabled');
+                $extensionsEnabled = json_decode($raw);
+
+                return new Less_Tree_Quoted('', in_array($extensionId->value, $extensionsEnabled) ? 'true' : 'false');
+            }
+        ];
+
+        $this->customFunctions = array_merge($coreFunctions, $this->customFunctions);
+
+        $compiler->setCustomFunctions($this->customFunctions);
```

Comment:
Why not move this logic to a singleton on the Frontend service provider? That's a better place to have config like this IMO.

Reply:
Done.